app/model/sdl/Abstract/Model.js

```diff
@@ -1164,8 +1249,30 @@ SDL.SDLModel = Em.Object.extend({
       SDL.InteractionChoicesView.cancelID = message.params.cancelID;
 
       if (message.params && message.params.choiceSet == null) {
-        FFW.UI.sendUIResult(SDL.SDLModel.data.resultCode.SUCCESS, 
-          message.id, 'UI.PerformInteraction');
+        imageList = [];
+        if(request.params.vrHelp) {
+          for(var i = 0; i < request.params.vrHelp.length; i++) {
+            var image = request.params.vrHelp[i].image;
+            if(image) {
+              imageList.push(image.value);
+            }
+          }
+        }
+                                                                        
+        var callback = function(failed) {
+          if(failed > 0) {
```

Comment:
Assuming `failed` is bool
```suggestion
          FFW.UI.sendUIResult(
              failed ? SDL.SDLModel.data.resultCode.WARNINGS : SDL.SDLModel.data.resultCode.SUCCESS,
              request.id, 
              request.method, 
              failed ? "Requested image(s) not found" : null
              );
```

Reply:
@AKalinich-Luxoft done in 0e61c5b